project/ProjectDependencies.scala

```diff
@@ -22,6 +22,8 @@ object ProjectDependencyVersions {
   val Log4jVersion = "2.5"
   val ProtobufVersion = "2.5.0"
   val SparkVersion = "1.6.1"
+  val VertxVersion = "3.0.0"
+  val ExampleVertxVersion = "3.3.2"
```

Comment:
Isn't that only an issue when using a `provided` configuration. If we force applications to use 3.3.2 this should be ok, unless there are breaking changes in Vert.x 3.x minor versions. Is this the case?